.github/workflows/ci.yml

```diff
@@ -653,6 +677,14 @@ jobs:
             ignore exclusion if: >-
               ${{ needs.build-pt.outputs.have_azure_secrets == 'true' }}
 
+          - suite: suite-gcs
+            config: default
+          - suite: suite-gcs
+            config: cdh5
```

Comment:
I suppose "suite-gcs x cdh5" is no longer needed https://github.com/trinodb/trino/commit/1ed0189c86346d3ca552c26b6887f37cd3bfa8ab